dialog/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# $Id$
+# Maintainer: David Mott <mott.david.j@gmail.com>
+
+pkgname=dialog
+pkgver=1.3_20160828
+pkgrel=1
+epoch=1
+pkgdesc="A tool to display dialog boxes from shell scripts"
+arch=('i686' 'x86_64')
+url="http://invisible-island.net/dialog/"
+license=('LGPL2.1')
+makedepends=('ncurses-devel')
+source=(ftp://invisible-island.net/$pkgname/$pkgname-${pkgver/_/-}.tgz)
+sha256sums=('453095abaec288bfbc1ca9faced917e17742cff1ea45ec46210071ac153562f9')
+
+build() {
+  cd "$srcdir/$pkgname-${pkgver/_/-}"
+
+  ./configure \
+    --prefix=/usr \
+    --enable-nls \
+    --with-ncursesw
+  make dialog.exe.exe
```

Comment:
yea its the most succinct way to port it that I found. its been a while since i looked at it, but memory serving me the build truncates everything after the last period and the build would fail.